.github/workflows/ci.yml

```
@@ -43,6 +43,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         MIX_ENV: test
         AUTH_API_KEY: ${{ secrets.AUTH_API_KEY }}
+        AUTH_URL: dwylauth.herokuapp.com
```

Now that auth in on Fly.io should the link be:

```suggestion
        AUTH_URL: auth.dwyl.com
```